bolt-modules/boltlib/spec/functions/apply_prep_spec.rb

```diff
@@ -76,6 +76,13 @@
       end
     end
 
+    it 'fails if version task is not found' do
```

Comment:
We expect this to happen because users may have used their own puppet_agent package?

Reply:
Yes. It's a flaw we'll need to work to address.